Web/Edubase.Web.UI/Areas/Governors/Models/GovernorsGridViewModel.cs

```diff
@@ -111,18 +112,18 @@ private void CreateGrids(GovernorsDetailsDto dto, IEnumerable<GovernorModel> gov
             foreach (var role in roles)
             {
                 var equivalantRoles = RoleEquivalence.GetEquivalentToLocalRole(role).Cast<int>().ToList();
-                var pluralise = !EnumSets.eSingularGovernorRoles.Contains(role);
+                //var pluralise = !EnumSets.eSingularGovernorRoles.Contains(role);
```

Comment:
This commented line conditionally flags if the role name should be plural or not, while the replacement sets this to permanently true (line 118).

If we're moving to having all roles plural on this page, let's remove this line?
Else let's uncomment this line and re-introduce the variable on line 118?

Reply:
Removed commented-out code. The logic as to whether a particular governor role should be pluralised has been moved to the GovernorRoleNameFactory

Reply:
Addendum - I had a misunderstanding here which @CathLass has kindly corrected me on: the boolean value is `pluraliseLabelIfApplicable`, and the conditional logic is within that method.